package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/dex-contracts",
-  "version": "0.1.3",
+  "version": "0.1.5-dev",
```

Comment:
It's pretty annoying having to publish versions to test this PR that are then public for the rest of time. Can we not use something like https://stackoverflow.com/questions/15806241/how-to-specify-local-modules-as-npm-package-dependencies to test this locally without publicly pushing new versions?

Can we please revert and delete those releases?